apps/nextjs/src/pages/index.tsx

```diff
@@ -110,15 +110,17 @@ const Home: NextPage = () => {
               ) : (
                 <div className="flex h-[40vh] justify-center overflow-y-scroll px-4 text-2xl">
                   <div className="flex w-full flex-col gap-4">
-                    {postQuery.data?.map((p) => {
-                      return (
-                        <PostCard
-                          key={p.id}
-                          post={p}
-                          onPostDelete={() => deletePostMutation.mutate(p.id)}
-                        />
-                      );
-                    })}
+                    {postQuery.data?.map(
+                      (p: { id: string; title: string; content: string }) => {
```

Comment:
Again, should be inferred.

Did yii ok u forget to run db:generate? Might have to restart the TS server afterwards too

Reply:
I think I didn't restart the TS server thats why I kept on seeing the squiggly lines. I have removed it rn